### ISSUE TITLE
error: variable ‘std::ifstream file’ has initializer but incomplete type

### ISSUE THREAD

**Issue by Ar-Ray-code** [Ar-Ray-code/YOLOX-ROS#11]
Report bug on Jetson（Ubuntu18）

```bash
yolox_tensorrt.cpp:16:28: error: variable ‘std::ifstream file’ has initializer but incomplete type
         std::ifstream file(path_to_engine, std::ios::binary);
```

**Comment by Ar-Ray-code** [comment]
Add `#include <fstream>` to solve this problem  in `yolox_ros_cpp/yolox_cpp/include/yolox_cpp/yolox_tensorrt.hpp`. I'll fix it.

**Comment by Ar-Ray-code** [comment]
fixed by #15
